### PR TITLE
It would be nice to have a button to move from one event to the next #589

### PR DESCRIPTION
**Pull request by valterbarros**
- Create new method on model to find next and previous event
- Cover both by test

Signed-off-by: ValterBarros <valterinside@gmail.com>

### CODE REVIEW

**Comment by valterbarros** [comment]
closes #589. Guys, I did create a part from backend and something is missing but how today is the last day from hacktorberfest I already up this pr to guarantee my t-shirt ;D. Trust me I will continue the work.

**Comment by kgf** [comment]
@valterbarros No problem, we understand. I'll put a `On hold` label on it, keep us updated. I commented in the issue, #589

File: spec/models/event_spec.rb [hunk 1]
```diff
@@ -72,4 +72,30 @@
...
+  describe '#next_event' do
+    let(:event_001) { create :event, start_time: "2017-11-02 03:36:00", end_time: "2017-11-04 03:36:00" }
+    let(:event_002) { create :event, start_time: "2017-11-03 03:36:00", end_time: "2017-11-06 03:36:00" }
+    let(:result) { event_001.next_event }
+    before do 
+      event_002
+    end
+
+    it 'should return event_002' do
+      expect(result).to eq event_002
```
Comment (kgf) [inline spec/models/event_spec.rb]: These tests would be much more valuable if you added an `event_000` and an `event_003`. Think if someone changed the code to get the last event, not the next. These tests would still pass.
Also, if you put this in a context, your `let` statements could happen once, eliminating the Code Climate complaint.

Comment (valterbarros) [reply spec/models/event_spec.rb]: Yeah, good.

**Comment by kgf** [comment]
@valterbarros I have merged this because it does advance issue #589 , but it does not close it, because it does not include the UI portion. Perhaps you would be willing to take that on as well ?

**Comment by valterbarros** [comment]
@kgf Yes I can do this

**Comment by kgf** [comment]
It would be nice to use blue buttons similar to the blue + on the events page.
Thanks for your help!
Kevin

**Pull request merged**